intelliserver/api/functions/data/index.js

```diff
@@ -0,0 +1,105 @@
+const express = require('express');
+const path = require('path');
+const axios = require('axios');
+const router = express.Router();
+const { ChatGPTInput, LLamaReplicateInput, LLamaSageInput, CohereInput } = require('intellinode');
+const { Chatbot, SupportedChatModels } = require('intellinode');
+const { USE_DEFAULT_KEYS } = require(path.join(global.__basedir, 'config'));
+
+const keys = {
+    'openai': process.env.OPENAI_API_KEY,
+    'replicate': process.env.REPLICATE_API_KEY,
+    'sagemaker': process.env.SAGEMAKER_API_KEY,
+    'cohere': process.env.COHERE_API_KEY
+};
+
+async function callSemanticSearchAPI(data) {
+    try {
+        const apiUrl = 'https://1cw9da6wl7.execute-api.us-east-2.amazonaws.com/semantic_search/';
```

Comment:
read the api from .env file with default url.

something like:
apiUrl  = os.environ.get('INTELLI_SEARCH_URL', 'https://1cw9da6wl7.execute-api.us-east-2.amazonaws.com/semantic_search/')

Reply:
feedback implemented